persistence-typed/src/main/scala/org/apache/pekko/persistence/typed/internal/BehaviorSetup.scala

```diff
@@ -107,14 +113,18 @@ private[pekko] final class BehaviorSetup[C, E, S](
 
   private var recoveryTimer: OptionVal[Cancellable] = OptionVal.None
 
+  val recoveryEventTimeout: FiniteDuration = persistence
```

Comment:
private please

Reply:
Can you add a `@since 1.1.3` annotation to any new public classes, variables or functions?

Reply:
Just double-checking - is this required for public members of classes marked with `@InternalApi` and `private[pekko]`?

Reply:
I'm not sure - this whole PR is looking like something we might need to stick in a future major release as opposed to an upcoming patch release. Leave the since annotations out for now.

Reply:
Understood. Is there any issue with creating a PR with this code in Akka repository? Wondering about any potential license conflicts, would not want interaction with BSL repositories voiding Pekko of this code.

Reply:
Feel free to submit this to Akka. You own this code and submitting it Akka does not change that. The problem will only arise if you collaborate on some of the code with a Lightbend employee and then take that collaboration and try to submit that to us.